index.js

```diff
@@ -18,4 +18,7 @@
  *
  */
 
-module.exports = exports = require('./lib/openshift-rest-client.js');
+module.exports = exports = {
+  OpenshiftClient: require('./lib/openshift-rest-client.js'),
```

Comment:
If you change the export in `./lib/openshift-rest-client.js` to be `OpenshiftClient` you could just eliminate the key on this line (minor nit, really)